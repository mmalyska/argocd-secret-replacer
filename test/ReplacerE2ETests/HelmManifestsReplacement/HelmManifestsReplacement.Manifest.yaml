---
# Source: argocd-secret-replacer-example/templates/secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: argocd-secret-replacer-chart-example-secret
data:
  sample-secret: PHNlY3JldDpzZWNyZXRLZXkxPg==
  sample-secret-quoted: "PHNlY3JldDpzZWNyZXRLZXkxPg=="
  sample-secret-multiline: b3B0aW9uczoNCiAgYWNjZXNzVG9rZW46IDxzZWNyZXQ6bXVsdGlsaW5lS2V5MT4NCiAgc2VjcmV0TmVlZGVkOiA8c2VjcmV0Om11bHRpbGluZUtleTI+
type: Opaque
---
# Source: argocd-secret-replacer-example/templates/configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: argocd-secret-replacer-chart-example-configmap
data:
  sample-secret: <secret:secretKey1>
  sample-secret-multiline: |
    options:
      accessToken: <secret:multilineKey1>
      secretNeeded: <secret:multilineKey2>
