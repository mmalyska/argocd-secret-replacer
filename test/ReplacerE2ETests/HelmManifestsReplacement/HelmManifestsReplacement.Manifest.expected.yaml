---
# Source: argocd-secret-replacer-example/templates/secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: argocd-secret-replacer-chart-example-secret
data:
  sample-secret: c2VjcmV0VmFsdWUx
  sample-secret-quoted: "c2VjcmV0VmFsdWUx"
  sample-secret-multiline: b3B0aW9uczoNCiAgYWNjZXNzVG9rZW46IHNlY3JldA0KICBzZWNyZXROZWVkZWQ6IHN1cGVyU2VjcmV0
type: Opaque
---
# Source: argocd-secret-replacer-example/templates/configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: argocd-secret-replacer-chart-example-configmap
data:
  sample-secret: secretValue1
  sample-secret-multiline: |
    options:
      accessToken: secret
      secretNeeded: superSecret
